The CUDA build fails with the error:
```
/tmp/bwibking/_work/1/s/src/HydroRichtmeyerMeshkov/test_hydro2d_rm.cpp:52:62: error: call of overloaded 'DistributionMapping(<brace-enclosed initializer list>)' is ambiguous
   52 |         amrex::DistributionMapping const localDistribution({0});
      |                                                              ^
/tmp/bwibking/_work/1/s/extern/amrex/Src/Base/AMReX_DistributionMapping.H:74:10: note: candidate: 'amrex::DistributionMapping::DistributionMapping(const amrex::BoxArray&, int)'
   74 |     explicit DistributionMapping (const BoxArray& boxes,
      |          ^~~~~~~~~~~~~~~~~~~
/tmp/bwibking/_work/1/s/extern/amrex/Src/Base/AMReX_DistributionMapping.H:72:10: note: candidate: 'amrex::DistributionMapping::DistributionMapping(amrex::Vector<int>&&)'
   72 |     explicit DistributionMapping (Vector<int>&& pmap) noexcept;
      |          ^~~~~~~~~~~~~~~~~~~
/tmp/bwibking/_work/1/s/extern/amrex/Src/Base/AMReX_DistributionMapping.H:71:10: note: candidate: 'amrex::DistributionMapping::DistributionMapping(const amrex::Vector<int>&)'
   71 |     explicit DistributionMapping (const Vector<int>& pmap);
      |          ^~~~~~~~~~~~~~~~~~~
```

--- a/src/BinaryOrbitCIC/binary_orbit.cpp
+++ b/src/BinaryOrbitCIC/binary_orbit.cpp
@@ -101,7 +101,8 @@ template <> void RadhydroSimulation<BinaryOrbit>::computeAfterTimestep()
 		amrex::Box const box(amrex::IntVect{AMREX_D_DECL(0, 0, 0)}, amrex::IntVect{AMREX_D_DECL(1, 1, 1)});
 		amrex::Geometry const geom(box);
 		amrex::BoxArray const boxArray(box);
-		amrex::DistributionMapping const dmap({0});
+		amrex::Vector<int> ranks({0}); // workaround nvcc bug
+		amrex::DistributionMapping const dmap(ranks);
 		analysisPC.Define(geom, dmap, boxArray);
 		analysisPC.copyParticles(*CICParticles);
 

--- a/src/BinaryOrbitCIC/binary_orbit.cpp
+++ b/src/BinaryOrbitCIC/binary_orbit.cpp
@@ -101,7 +101,7 @@ template <> void RadhydroSimulation<BinaryOrbit>::computeAfterTimestep()
 		amrex::Box const box(amrex::IntVect{AMREX_D_DECL(0, 0, 0)}, amrex::IntVect{AMREX_D_DECL(1, 1, 1)});
 		amrex::Geometry const geom(box);
 		amrex::BoxArray const boxArray(box);
-		amrex::Vector<int> ranks({0}); // workaround nvcc bug
+		amrex::Vector<int> const ranks({0}); // workaround nvcc bug
 		amrex::DistributionMapping const dmap(ranks);
 		analysisPC.Define(geom, dmap, boxArray);
 		analysisPC.copyParticles(*CICParticles);

--- a/src/HydroRichtmeyerMeshkov/test_hydro2d_rm.cpp
+++ b/src/HydroRichtmeyerMeshkov/test_hydro2d_rm.cpp
@@ -49,7 +49,7 @@ template <> void RadhydroSimulation<RichtmeyerMeshkovProblem>::computeAfterTimes
 	// copy all FAB data to a single FAB on rank zero
 	amrex::Box const domainBox = geom[0].Domain();
 	amrex::BoxArray const localBoxes(domainBox);
-	amrex::Vector<int> ranks({0}); // workaround nvcc bug
+	amrex::Vector<int> const ranks({0}); // workaround nvcc bug
 	amrex::DistributionMapping const localDistribution(ranks);
 	amrex::MultiFab state_mf(localBoxes, localDistribution, ncomp_cc, 0);
 	state_mf.ParallelCopy(state_new_cc_[0]);

--- a/src/HydroRichtmeyerMeshkov/test_hydro2d_rm.cpp
+++ b/src/HydroRichtmeyerMeshkov/test_hydro2d_rm.cpp
@@ -49,7 +49,8 @@ template <> void RadhydroSimulation<RichtmeyerMeshkovProblem>::computeAfterTimes
 	// copy all FAB data to a single FAB on rank zero
 	amrex::Box const domainBox = geom[0].Domain();
 	amrex::BoxArray const localBoxes(domainBox);
-	amrex::DistributionMapping const localDistribution({0});
+	amrex::Vector<int> ranks({0}); // workaround nvcc bug
+	amrex::DistributionMapping const localDistribution(ranks);
 	amrex::MultiFab state_mf(localBoxes, localDistribution, ncomp_cc, 0);
 	state_mf.ParallelCopy(state_new_cc_[0]);
 

